react-client/src/IndexDesc.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import MainDesc from './MainDesc';
+import sampleData from '../sampleData';
+
+class Index extends React.Component {
```

Comment:
name of file does not match the name of the component...might be easier to match them up just for consistency